haml.md

```diff
@@ -0,0 +1,114 @@
+Working with [ERB](https://www.tutorialspoint.com/ruby-on-rails/rails-and-html-erb.htm) can get the job done, but there are fantasic alternatives to achieve the same result with much less code. 
+
+Input [HAML](https://haml.info/). Haml (HTML abstraction markup language) is based on one primary principle: markup should be beautiful. It’s not just beauty for beauty’s sake either; Haml accelerates and simplifies template creation down to veritable haiku.
```

Comment:
This feel a little sales-y. Like, this could be printed on a hand towel too. 😆 

Kidding aside: I could feel the tone change from "Isaac is writing this" to "this is boilerplate that was copy-n-pasted".

Copy-n-paste is 100% okay. But maybe call it out?

```suggestion
Input [HAML](https://haml.info/). From their website:

> Haml (HTML abstraction markup language) is based on one primary principle: markup should be beautiful. It’s not just beauty for beauty’s sake either; Haml accelerates and simplifies template creation down to veritable haiku.

At BNB, we use HAML's indentation-based markup to manage the nesting our HTML nodes for us. No more managing of closing `</div>` tags everywhere!
```

Or something.